### PR TITLE
Maya: Validate missing instance attributes

### PR DESCRIPTION
**Pull request by tokejepsen**
## Brief description
Validate missing instance attributes.

## Description
New attributes can be introduced as new features come in. Old instances will need to be updated with these attributes for the documentation to make sense, and users do not have to recreate the instances.

## Testing notes:
1. Make an instance in Maya.
2. Delete an attribute from the instance's objectset.
3. Publish.

### CODE REVIEW

**Review by LiborBatek — APPROVED** [review]
Followed the testing steps:

1) Deleted ```review width``` and ```review height``` from ```review``` instance

2) Validator correctly validated it that missing attributes on the instance ```review```

image


3) Repair action correctly added those missing attributes 

image


4)Succesfully published

image

**Comment by BigRoy** [comment]
Clever.

I assume this validator might be obsolete with the new publisher. Does anyone know?

File: openpype/hosts/maya/plugins/publish/validate_instance_attributes.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import pyblish.api
+from openpype.pipeline.publish import (
+    ValidateContentsOrder, PublishValidationError, RepairAction
+)
+from openpype.pipeline import discover_legacy_creator_plugins
+from openpype.hosts.maya.api.lib import imprint
+
+
+class ValidateInstanceAttributes(pyblish.api.InstancePlugin):
+    """Validate Instance Attributes."""
```
[the remark this answers is not in the document]
Comment (mkolar) [reply openpype/hosts/maya/plugins/publish/validate_instance_attributes.py:12]: Agreed, this one especially is a tiny bit cryptic, so a tiny explanation akin to the PR description would be great here

**Comment by mkolar** [comment]
> Clever.
> 
> I assume this validator might be obsolete with the new publisher. Does anyone know?

Yes I believe this would be obsolete as soon as maya conversion to new publisher is finalised

File: openpype/hosts/maya/plugins/publish/validate_instance_attributes.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tokejepsen) [reply openpype/hosts/maya/plugins/publish/validate_instance_attributes.py]: Done.

**Review by kalisp — APPROVED** [review]
Code looks good.

**Pull request merged**